src/transport/SessionManager.cpp

```diff
@@ -292,13 +322,8 @@ CHIP_ERROR SessionManager::SendPreparedMessage(const SessionHandle & sessionHand
         char addressStr[Transport::PeerAddress::kMaxToStringSize];
         multicastAddress.ToString(addressStr, Transport::PeerAddress::kMaxToStringSize);
 
-        ChipLogProgress(Inet,
-                        "Sending %s msg %p with MessageCounter:" ChipLogFormatMessageCounter " to %d"
-                        " at monotonic time: " ChipLogFormatX64
-                        " msec to Multicast IPV6 address : %s with GroupID of %d and fabric index of %x",
-                        "encrypted group", &preparedMessage, preparedMessage.GetMessageCounter(), groupSession->GetGroupId(),
-                        ChipLogValueX64(System::SystemClock().GetMonotonicMilliseconds64().count()), addressStr,
-                        groupSession->GetGroupId(), groupSession->GetFabricIndex());
+        ChipLogProgress(Inet, "(G) Sending msg " ChipLogFormatMessageCounter " to Multicast IPV6 address '%s'",
```

Comment:
The (G) had the string getting in the session type. Any reason to keep the GetSessionTypeString if this code is overriding that?

Reply:
It's used in the TX and RX prints, in both `ExchangeMgr` and `SessionManager`. As per our discussion, have added a separate `GetSessionTypeString()` standalone function in `Session.h` and removed all the conditional APIs in the various Session subclasses.